TAO/TAO_IDL/driver/drv_preproc.cpp

```diff
@@ -1354,7 +1354,14 @@ DRV_pre_proc (const char *myfile)
           throw Bailout ();
         }
 
-      cpp_options.set_handles (ACE_INVALID_HANDLE, fd);
+      if (cpp_options.set_handles (ACE_INVALID_HANDLE, fd) == -1)
+        {
+          ACE_ERROR ((LM_ERROR,
+                      "%s: cannot set stdout for child process: %p\n",
+                      ACE_TEXT_CHAR_TO_TCHAR (idl_global->prog_name ())));
```

Comment:
Thanks, could you at the same moment check the other occurences of prog_name() logging? Then all are consistent.